mutation-testing/shell-scripts/append-match-package.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# the append-match-package.sh
+## goes through each line in the output and based on the package ( first element before /)
+### verifies the line with the other lines in that specific folder
+#### in our case folder_name == package_name
+
+
+# goes through each PR file line by line
+# for each first_element/the_rest_of_the_line goes through it
+## search in that specific folder on all 4 files
+## if it is matchy, remove it from that file
+## based on the file it was taken from, append it to the same file in the STABLE folder
+
```

Comment:
It's considered good practice to add `set -euo pipefail` to all bash scripts. This will cause cause failures on certain errors, rather continuing to run and producing bad results